Userland/Applications/DisplaySettings/MonitorWidget.cpp

```diff
@@ -58,7 +58,7 @@ bool MonitorWidget::set_wallpaper(String path)
     return true;
 }
 
-String MonitorWidget::wallpaper()
+String& MonitorWidget::wallpaper()
```

Comment:
What is the purpose of returning a mutable reference to `m_desktop_wallpaper_path`?

Reply:
None at all I'd say! Both `m_desktop_wallpaper_path` and `m_desktop_wallpaper_mode` getters return const references now